code/modules/assembly/flash.dm

```diff
@@ -39,8 +40,10 @@
 	return FLASH_USE
 
 /obj/item/flashbulb/weak
-	name = "weakened flashbulb"
-	charges_left = 4
+	name = "cyborg flashlight bulb"
+	desc = "a lamp intended for use within a cyborg eye."
+
+	charges_left = 1
```

Comment:
/weak is for weak bulbs, not cyborg eyes specifically. Make a new subtype for cyborg eyes, this is used elsewhere I believe

Reply:
Quoting my response from a bit further down:

> There are six weak flashes located on every map in the robotics lab at roundstart. My understanding is these are explicitly for making cyborgs with. Outside of these six, the only way to get more is by printing them.

> They are not implemented elsewhere at all. Creating a new subtype is completely pointless because I would be replacing these six spawns and printed version with cyborg eyes leaving no references to the weak flash

Just like with the flashes, making a new object is totally pointless because I am going to replace *every* instance of weak flash with cyborg eyes. The only reason weak flashes currently existed was so cyborgs could be built without giving roboticists access to full-powered flashes.

Reply:
Then rename the typepath to /obj/item/flashbulb/cyborg and completely delete /obj/item/flashbulb/weak if you are sure that it is not used anywhere else and serves no purpose.